tweetCrawler/models.py

```diff
@@ -18,3 +18,7 @@ class Tweet(models.Model):
 
     searches = models.ManyToManyField(SearchParameters)
     google = models.ManyToManyField(GoogleResultOfficial)
+
+    @property
+    def get_node_id(self):
+        return 'google' + str(self.id)
```

Comment:
shouldn't it be 'tweet' instead of 'google'?

Reply:
Yes, it should